samples/bluetooth/llpm/README.rst

```diff
@@ -0,0 +1,313 @@
+.. _ble_llpm:
+
+Bluetooth: LLPM
+###############
+
+The Bluetooth Low Latency Packet Mode (LLPM) sample uses the :ref:`latency_readme` to showcase the LLPM proprietary Bluetooth extension from Nordic Semiconductor.
+You can use it to determine the transmission latency of LLPM-enabled connections, or to compare with different connection parameters and check their influence on the results.
+
+
+Overview
+********
+
+The LLPM is designed time-sensitive transmissions, in which time synchronization is important and interface response time is critical for the user.
+For example, for reality headsets or gaming mouse and keyboard.
+
+See the following subsections for a description of the key LLPM elements.
+
+LLPM connection interval (1 ms)
+   The connection interval defines how often the devices must listen on the radio.
+   The LLPM introduces the possibility to reduce the connection interval below what is supported in BLE.
+   The lowest interval supported is 1 ms for one link.
+
+Physical layer (PHY)
+   Starting with Bluetooth 5, the over-the-air data rate in Bluetooth Low Energy supports 2 Ms/s (mega symbol per second), which allows for faster transmission.
+   The LLPM connection interval is only supported on *LE 2M PHY*.
+   Otherwise, the BLE controller will deny the request command.
+
+QoS connection event reports
+   When reports are enabled, one report will be generated on every connection event.
+   The report gives information about the quality of service of the connection event.
+   The values in the report are used to describe the quality of links.
+   For parameter descriptions, see :cpp:enum:`hci_vs_evt_qos_conn_event_report_t` (in :file:`ble_controller_hci_vs.h`).
+
+Transmission latency
+   The definition of the latency used in this example counts the time interval from the sender's application to the GATT service of the receiver.
+   It demonstrates the performance of an LLPM-enabled connection that the receiver will receive the data approximately every 1 ms.
+
+GATT Latency Service
+   To measure the transmission latency from application layers, a GATT Latency service `BT_UUID_LATENCY` is included to compute the time spent.
+   When the sender writes its timestamp to the `BT_UUID_LATENC_CHAR` characteristic of the receiver, the Latency service of the receiver will automatically reply back.
+   Whenever the sender receives a response, it will use its present time and the corresponding timestamp written before to estimate the round-trip time (RTT) of a writing characteristic procedure (see `Bluetooth Core Specification`_: Vol 3, Part F, 3.4.5 Writing attributes).
+
+.. list-table:: GATT Attributes
+   :header-rows: 1
+
+   * - Type
+     - UUID
+     - Property
+   * - Primary service
+     - BT_UUID_LATENCY
+     - Read only
+   * - Characteristic
+     - BT_UUID_LATENCY_CHAR
+     - BT_GATT_CHRC_WRITE, BT_GATT_PERM_WRITE
+
+This sample transmits data between two boards to measure the transmission latency in between.
```

Comment:
In the sample documentation we mention 3 times how the latency is computed. I would say that keeping only the MSC is sufficient.